cmd/noms/noms_show.go

```diff
@@ -49,6 +51,10 @@ func runShow(args []string) int {
 		return 0
 	}
 
+	if showRaw && showStats {
+		fmt.Fprintln(os.Stderr, "--raw and --stats are mutually exclusive")
```

Comment:
hmm.. does kinda seem like it should return non-zero, but i'm gonna be lazy and consistent with surrounding code.